awx/main/migrations/0126_executionenvironment_container_options.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.16 on 2021-01-27 22:31
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0125_more_ee_modeling_changes'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='executionenvironment',
+            name='container_options',
+            field=models.CharField(choices=[('always', 'Always pull container before running.'), ('missing', 'No pull option has been selected'), ('never', 'Never cull container before running')], default='missing', help_text='Pull image before running?', max_length=1024),
```

Comment:
```suggestion
            field=models.CharField(choices=[('always', 'Always pull container before running.'), ('missing', 'No pull option has been selected'), ('never', 'Never pull container before running')], default='missing', help_text='Pull image before running?', max_length=1024),
```

Reply:
that would have meant something very different 😅